lib/react-popout.jsx

```diff
@@ -87,9 +87,16 @@ export default class PopoutWindow extends React.Component {
     }
 
     componentWillReceiveProps(newProps) {
-        if (newProps.title !== this.props.title && this.state.popoutWindow) {
-
-            this.state.popoutWindow.document.title = newProps.title;
+        if (this.state.popoutWindow) {
+            if (this.props.title !== newProps.title) {
+                this.state.popoutWindow.document.title = newProps.title;
+            }
+            if (this.props.url !== newProps.url) {
+                this.state.popoutWindow.postMessage({
+                    type: 'receiveSco',
```

Comment:
Should we standardize the types for these?

Reply:
Yes. Down the road a ways, once we know what a few more of them are.